docs/en/observability/synthetics-create-test.asciidoc

```diff
@@ -207,6 +207,8 @@ with the corresponding https://playwright.dev/docs/api/class-browsercontext[`Bro
 via <<elastic-synthetics-command, `--playwright-options`>> or in the
 <<synthetics-configuration, `synthetics.config.ts` file>>.
 
+See a full example using the `request` parameter in the https://github.com/elastic/synthetics-demo/blob/main/advanced-examples/journeys/api-requests.journey.ts[Elastic Synthetics demo repository].
```

Comment:
"full example using" is grammatically ambiguous...could be read as if the request parameter is used to see the example. Also the accessibility guidelines tell us to avoid using "see" because it's ableist (not sure I agree, but that's the guideline).

How about saying something like this (to match the comment in the example file):

For a full example that shows how to use the `request` object, refer to  the....

(Not sure whether you should use `request` parameter or `request` object but it would be good for the text here to match the comment in the example.)